docs/running-notebooks/opening-sessions/index.md

```diff
@@ -14,6 +14,12 @@ It could take several minutes to start the interactive session.
 Please note that it will be automatically closed after 7 days of inactivity.
 ```
 
+By clicking on the link you will have access to a Jupyter notebook interface:
+
+![jupyter-notebook](../../images/interactive-session-jupyter-notebook.png){.screenshot-browser-mockup}
+
+Keep in mind that starting the notebook might take a minute, if you see "Service unavailable", just wait and retry.
+
 !!! warning
```

Comment:
The Warning box also disrupts the reading flow. I would do put it at the end of the CLI chapter, i.e. right before the "Open from web interface" section. And I would add "Please note that it will be automatically closed after 7 days of inactivity" to the last example (using custom image) as well. WDYT?